concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -27,17 +27,54 @@ local buildpackagejob = {
     { load_var: 'commit-sha', file: '%s/.git/ref' % tl.package },
     {
       task: 'generate-timestamp',
-      file: 'guest-test-infra/concourse/tasks/generate-timestamp.yaml',
+      config: {
+        platform: 'linux',
+        image_resource: {
+          type: 'registry-image',
+          source: { repository: 'bash' },
+        },
+        outputs: [{ name: 'timestamp' }],
+        run: {
+          path: '/usr/local/bin/bash',
+          args: [
+            '-c',
+            'timestamp=$((${EPOCHREALTIME/./}/1000)); echo $(($timestamp/1000)) | tee timestamp/timestamp; echo\n  $timestamp | tee timestamp/timestamp-ms',
+          ],
+        },
+      },
     },
     { load_var: 'start-timestamp-ms', file: 'timestamp/timestamp-ms' },
-    {
-      task: 'get-github-token',
-      file: 'guest-test-infra/concourse/tasks/get-github-token.yaml',
-    },
     {
       task: 'generate-package-version',
-      file: 'guest-test-infra/concourse/tasks/generate-package-version.yaml',
-      input_mapping: { repo: tl.package },
+      config: {
+        platform: 'linux',
+        image_resource: {
+          type: 'registry-image',
+          source: { repository: 'alpine/git' },
+        },
+
+        inputs: [{ name: tl.package, path: 'repo' }],
+        outputs: [{ name: 'package-version' }],
+
+        run: {
+          path: 'ash',
```

Comment:
did you mean `bash` or `sh` here?

Reply:
no this is intended to be 'ash', these args are copied from the original task 
https://github.com/GoogleCloudPlatform/guest-test-infra/blob/master/concourse/tasks/generate-package-version.yaml

Reply:
ok